config/core/rootfs-images.yaml

```diff
@@ -31,6 +31,14 @@ file_systems:
     ramdisk: bullseye-cros-ec/20230623.0/{arch}/rootfs.cpio.gz
     root_type: ramdisk
     type: debian
+  debian_bullseye-disk:
```

Comment:
OK we'll need to get today's production update done with one rootfs image update (`bullseye-gst-fluster`) and then this might need to be rebased.  Also the script used to automatically update and check the rootfs URLs will need to take the `disk` (or `diskimage`) rootfs variants into account.